src/transformJS.js

```diff
@@ -4,7 +4,10 @@ const babel = require('@babel/core');
 // addSetupNode adds the setup() call to the beginning of the script.
```

Comment:
Maybe call this file `transformScript.js` for consistency?

Reply:
Yes! Such a good name. I was not happy with the one I picked